src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -115,4 +115,17 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
       }
     }
   }
+
+  it should "error when running a workflowless WDL" {
+    DataAccess.withDataAccess { dataAccess =>
```

Comment:
maybe import `DataAccess._` so "data access" isn't used for three consecutive words?  :smile: